common/lib/xmodule/xmodule/js/spec/video/video_speed_control_spec.js

```diff
@@ -38,21 +38,11 @@
                         expect($(link)).toHaveData(
                             'speed', state.speeds[index]
                         );
-                        expect($(link).find('a').text()).toBe(
+                        expect($(link).find('.speed-link').text()).toBe(
```

Comment:
What is the difference between `.speed-link` and `.speed-button` (that class also exists already)?